.github/workflows/ci.yml

```diff
@@ -0,0 +1,288 @@
+name: Rust
+on:
+  push:
+    branches:
+      - master
+      - ci # TODO: remove later
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+  pull_request:
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
```

Comment:
Seems I forget to remove this option while adopting this rules from another project.